guides/source/routing.md

```diff
@@ -1165,6 +1165,43 @@ video = Video.find_by(identifier: "Roman-Holiday")
 edit_video_path(video) # => "/videos/Roman-Holiday/edit"
 ```
 
+Breaking up *very* large route file into multiple small ones:
+-------------------------------------------------------
+
+If you work in a large application with hundreds if not thousands of routes,
```

Comment:
I'd say the cut-off is thousands. We have 500+ at Basecamp, and the split is def not worth it there.

Reply:
We are including different routes on boot, based on some environment variables (like hostname), so for us it's not the total size, but the general separation of different customer routes which is important for us. So there are other factors too to consider, not only the size :)